sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
```

Comment:
Don't forget to add this sucker before the "Features Added" header:

```markdown
### Acknowledgments

Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:

- Daniel Marbach _([GitHub](https://github.com/danielmarbach))_
```

It's not much but we can at least highlight your efforts rather than burying them inline.  😄